flowclient/flowclient/client.py

```diff
@@ -935,3 +935,38 @@ def flows(
         "to_location": to_location,
         "aggregation_unit": aggregation_unit,
     }
+
+
+def unique_subscriber_counts(
+    start_date: str,
+    stop_date: str,
+    aggregation_unit: str,
+) -> dict:
+    """
+    Return query spec for unique subscriber counts
+
+    Parameters
+    ----------
+    start_date : str
+        ISO format date of the first day of the count, e.g. "2016-01-01"
+    end_date : str
+        ISO format date of the day _after_ the final date of the count, e.g. "2016-01-08"
+    aggregation_unit : str
+        Unit of aggregation, e.g. "admin3"
+
+    Returns
+    -------
+    dict
+        Dict which functions as the query specification
+
+    Example
+    -------
+    >>> unique_subscriber_counts( { "start_date" : "2019-03-20", "end_date" : "2019-03-31", "aggregation_unit" : "admin3" } )
+    [ ]  # TODO - Fill this in
```

Comment:
Accidentally forgot to delete this TODO item?

Reply:
I wasn't sure what goes there. Is there a systematic way to find out? I assume I can try actually running the code and extracting the relavent string from the output.

Reply:
Yes, you'd need to actually run it and copy & paste the output.

Note that in order to be able to successfully run it, you will need to change the dates to something between `2016-01-01` and `2016-01-07` because the data in the test database is between those dates.